themes/openy_themes/openy_rose/templates/node/node--branch--teaser.html.twig

```diff
@@ -99,18 +100,13 @@
   {{ content.field_location_state }}
   {{ content.field_location_area }}
 
-  {% if display_submitted %}
-    <footer class="node__meta">
-      {{ author_picture }}
-      <div{{ author_attributes.addClass('node__submitted') }}>
-        {% trans %}Submitted by {{ author_name }} on {{ date }}{% endtrans %}
-        {{ metadata }}
-      </div>
-    </footer>
-  {% endif %}
-
   <div{{ content_attributes.addClass('node__content') }}>
-    {{ content|without('field_location_area', 'field_location_state') }}
+    {{ content.field_location_address }}
+    {{ content.field_location_phone }}
+    {% if content.field_location_directions.0['#title'] %}
```

Comment:
Please set this variable at the beginning of the template, and then check it here. We should never use array attributes in the template itself. Only if definitions of variables.

The same for variables below.